paddle/gserver/layers/SequenceLastInstanceLayer.cpp

```diff
@@ -25,6 +25,9 @@ namespace paddle {
  * Input: a sequence
  * If SequenceLevel = kNonseq:
  *   Output: a sequence containing only the last instance of the input sequence
+ *   If stride_ > 0:
+ *      Output: a shorten sequence containing several last instances of the
+ *              input sequence with stride window.
```

Comment:
Output: a shorten sequence. The operation of getting last instance of a sequence is independently performed on every slice of the input sequence, that is obtained by sliding a window with the window size set to stride_.